README.md

```diff
@@ -70,28 +70,29 @@ For builds targeting RISC-V, ARM32, or ARM64 hardware, see the [Other platforms
 
 ### ESP-IDF
 
-The VM has a requirement to ESP-IDF, both for Linux and ESP32 builds (for Linux it's for the [Mbed TLS](https://www.trustedfirmware.org/projects/mbed-tls/) implementation).
+The Toit VM has a requirement for the [Espressif IoT Development Framework](https://idf.espressif.com/), both for Linux and ESP32 builds (for Linux it's for the [Mbed TLS](https://www.trustedfirmware.org/projects/mbed-tls/) implementation).
 
 We recommend you use Toitware's [ESP-IDF fork](https://github.com/toitware/esp-idf) that comes with a few changes:
 
 * Custom malloc implementation.
 * Allocation-fixes for UART, etc.
 * LWIP fixes.
 
+The fork's repository has been added as a submodule reference to this repository, so doing a recursive submodule init & update will establish everything nedded:
+
 ``` sh
-git clone https://github.com/toitware/esp-idf.git
-pushd esp-idf/
-git checkout patch-head-4.3-3
 git submodule update --init --recursive
-popd
+
 ```
 
-Remember to add it to your ENV as `IDF_PATH`:
+For the build to succeed, you will need to add its path to your ENV as `IDF_PATH`:
 
 ``` sh
-export IDF_PATH=...
+export IDF_PATH=`pwd`/third_party/esp-idf
```

Comment:
I would make this the default, if IDF_PATH isn't set, but we can do that in a follow-up PR.

Reply:
The newer posix way seems to be to use `$()` instead of backticks.
```suggestion
export IDF_PATH=$(pwd)/third_party/esp-idf
```